src/ui/page/error/renderingError/renderingError.scss

```diff
@@ -0,0 +1,134 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-rendering-error-page-main {
+  @include with-theme() {
+    height: 100%;
+
+    &.dark {
+      background: center / cover url('../../../asset/images/rendering-error-dark.webp');
```

Comment:
Are absolute paths working for background url?

Reply:
Just checked, yes they are! Thanks!